### PR TITLE
misc(query): update logicalPlan utility methods

### PR DESCRIPTION
**Pull request by tjackpaul**
**Pull Request checklist**

- [x] The commit(s) message(s) follows the contribution [guidelines](CONTRIBUTING.md) ?
- [x] Tests for the changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?

### CODE REVIEW

File: query/src/test/scala/filodb/query/LogicalPlanSpec.scala [hunk 1]
```diff
@@ -145,8 +171,112 @@ class LogicalPlanSpec extends FunSpec with Matchers {
 
     val binaryJoin = BinaryJoin(lhs, DIV, OneToOne, rhs)
 
-    val res = LogicalPlan.getLabelValueFromLogicalPlan(binaryJoin, "instance")
-    res.get.shouldEqual(Seq("Inst-0", "Inst-1"))
+    val res = LogicalPlan.getColumnValues(binaryJoin, "instance")
+    res.shouldEqual(Set("Inst-0", "Inst-1"))
+  }
+
+  it("should sort LabelValueOperators when only one group is present") {
```
Comment (vishramachandran) [inline query/src/test/scala/filodb/query/LogicalPlanSpec.scala:178]: LabelValueOperators is removed?

File: query/src/test/scala/filodb/query/LogicalPlanSpec.scala [hunk 2]
```diff
@@ -145,8 +171,112 @@ class LogicalPlanSpec extends FunSpec with Matchers {
...
+        cfSet.column.shouldEqual("instance")
+        cfSet.filter.operatorString.shouldEqual("!~")
+        cfSet.filter.valuesStrings shouldEqual(Set("Inst-1"))
+      } else {
+        fail("invalid entry in column filter sequence " + cfSet)
+      }
+    }
   }
 
+  it("should have equal hashcode for identical LabelValueOperatorGroups") {
```
Comment (vishramachandran) [inline query/src/test/scala/filodb/query/LogicalPlanSpec.scala:267]: LabelValueOperatorGroups is removed?

File: query/src/test/scala/filodb/query/LogicalPlanSpec.scala (hunk 1 above)
Comment (tjackpaul) [reply query/src/test/scala/filodb/query/LogicalPlanSpec.scala:178]: yes, will update this test case

Comment (tjackpaul) [reply query/src/test/scala/filodb/query/LogicalPlanSpec.scala]: this is taken care of

File: query/src/main/scala/filodb/query/LogicalPlan.scala [hunk 3]
```diff
@@ -353,53 +349,46 @@ object LogicalPlan {
...
-  def getLabelValueFromGroup(labelName: String, group: LabelValueOperatorGroup): Seq[String] = {
-    group.labelValueOperators.flatMap(lops => {
-      lops.columnName.equals(labelName) match {
-        case true => lops.value
-        case false => Seq()
+  def getColumnValues(columnFilters: Set[ColumnFilter], labelName: String): Set[String] = {
+    columnFilters.flatMap(cFilter => {
+      cFilter.column == labelName match {
+        case true => cFilter.filter.valuesStrings.map(_.toString)
+        case false => Seq.empty
```
Comment (TanviBhavsar) [inline query/src/main/scala/filodb/query/LogicalPlan.scala:368]: Set.empty?

Comment (tjackpaul) [reply query/src/main/scala/filodb/query/LogicalPlan.scala:368]: Inside the map it is returning Seq. And flatmap will convert it to Set

**Pull request merged**